src/wasm/wasm-s-parser.cpp

```diff
@@ -1845,7 +1834,7 @@ Expression* SExpressionWasmBuilder::makeConst(Element& s, Type type) {
   }
 
   auto ret = allocator.alloc<Const>();
-  Type lane_t = stringToLaneType(s[1]->str().str);
+  Type lane_t = stringToLaneType(s[1]->str().str.data());
```

Comment:
Yes, that would be a good idea. I'll add a TODO.

Reply:
Although it's probably not worth spending too much time updating this parser since we're working on a replacement that already uses string_view extensively.

Reply:
Yeah then I agree it's not worth it.